integration-tests/http/__tests__/returns/returns.spec.ts

```diff
@@ -973,6 +973,7 @@ medusaIntegrationTestRunner({
               items: [
                 expect.objectContaining({
                   received_quantity: 2,
+                  damaged_quantity: 1,
```

Comment:
question: so in this case, the user receives 3 items: 2 regular and one damaged?

Reply:
make sense, so you can always base on received quantity to not go over it for example 👍